artemis/modules/DarkWebMonitor.py

```diff
@@ -0,0 +1,46 @@
+import requests
+from bs4 import BeautifulSoup
+
+class DarkWebMonitor:
+    def __init__(self):
+        self.base_url = "https://exampledarkweb.onion"
```

Comment:
do you have a concrete dark web site(s) to look for results?

Reply:
I am still working on that. This is the skeleton of the module I am thinking of building in the summer.

Reply:
Sure, let me know if you have concrete examples of sites to monitor and things to download from there!

Reply:
Do we need to get accepted to the program in order to get the pull request merged ?

Reply:
nope! But in order to get this merged you need to provide a pull request that is not a mock/skeleton - in this case, queries a real data source, is consistent with Artemis module API and displays data in the UI properly